### PR TITLE
Changed asciidoctorj and asciidoctorj-diagram to latest working versions.

### PR DESCRIPTION
**Pull request by bshambaugh**
The program was unable to build, so I looked in the maven repository for the latest versions of asciidoctorj* .

### CODE REVIEW

**Comment by bshambaugh** [comment]
I was able to launch the jvm/bin/mmadt.sh , but if there is something that this broke, I do not know it yet.

**Comment by okram** [comment]
Downgrading to older versions seems an awkward solution to the problem. Moreover, the versions of asciidoctorj main and diagrams are no longer synchronized. Have you built the docs to see if this causes problems with doc rendering?

**Comment by okram** [comment]
I just upgraded to the latest version of asciidoctorj (2.4.0 -- released a few hours ago). I removed the asciidoctorj-diagram dependency and built the docs. they render as expected. seems 2.4.0 includes diagramming in it.

**Comment by bshambaugh** [comment]
This is better. I didn't see 2.4.0. Moreover, I realized that I didn't build the docs.  Thanks. Agreed. What you went ahead with was optimal.

for the record, slack rant. 
"Hmm... I was using 3.2.0 for asciidoctorj . 2.4.0 was the latest version of asciidoctorj-diagram. I realized last night I didn't test the build with asciidoctor... just the jvm. changing the build.sbt script got the computer to stop complaining. (edited) 
4:37
realizing this, I'm looking at how the asciidoctor build part works
4:38
tl;dr...3.2.0 is in the maven repo. investigating.
4:40
oh nvm...I can't count....looking at https://repo1.maven.org/maven2/org/asciidoctor/asciidoctorj/2.4.0/"
